zonemaster/zonemaster-cli
# Add --hints option

## Purpose

Allow the user to override the built in root hints.

## Context

This change depends on zonemaster/zonemaster-engine#1134.

## Changes

A new `--hints=FILENAME` option is added.

## How to test this PR

1. Set up a custom root zone.
2. Create a root hints file (named.custom-root) for the custom root zone.
3. Run some zonemaster-cli tests with `--hints named.custom-root` and make sure it works as expected.
   * Make sure that zonemaster/zonemaster-engine#1134 is included in the Zonemaster::Engine being used.
